sympy/stats/joint_rv.py

```diff
@@ -284,6 +284,8 @@ def symbols(self):
         return set([rv.pspace.symbol for rv in rvs])
 
     def pdf(self, x, evaluate=False):
+        if not hasattr(x, '__iter__'):
```

Comment:
why this `hasattr`?

Reply:
Maybe the joint pdf should simply be defined with `pdf(self, *x)`.

Reply:
Should I remove the evaluate flag, then?

Reply:
I would say that it's best to keep the API compatible with the rest of the module. So yes, just *x